draft-ietf-jose-pqc-kem.md

```diff
@@ -217,8 +223,7 @@ The key derivation for JOSE is performed using the KMAC defined in NIST SP 800-1
    *  K: the input key-derivation key. In this document this is the initial shared secret (SS') outputted from the 
       kemEncaps() or kemDecaps() functions.
 
-   *  X: JOSE context-specific data defined in Section 4.6.2 of {{?RFC7518}}, i.e., concat(AlgorithmID, PartyUInfo, PartyVInfo, 
-      SuppPubInfo, SuppPrivInfo).
+   *  X: A subset of the JOSE context-specific data defined in Section 4.6.2 of {{RFC7518}}, i.e., concat(AlgorithmID, SuppPubInfo, SuppPrivInfo), is used. The fields PartyUInfo and PartyVInfo are excluded. PartyUInfo is omitted because sender authentication is not available in PQ KEMs. PartyVInfo is excluded because the recipient's identity is already bound to the public key used for encapsulation, making its inclusion redundant. If mutually known private information is to be included, both the sender and the recipient MUST agree out-of-band to include it as SuppPrivInfo in the key derivation function, as defined in {{NIST.SP.800-56Ar3}}. 
```

Comment:
What does `AlgorithmID` stand for here? Is that e.g. `MLKEM512` in case of Direct Key Agreement and `MLKEM512-AES128KW` in case of Key Agreement with Key Wrapping?